Samples~/MarigoldUnityApp.cs

```diff
@@ -0,0 +1,157 @@
+﻿using UnityEngine;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using MarigoldSimpleJSON;
+using MarigoldSDK;
+
+public class MarigoldUnityApp : MonoBehaviour {
```

Comment:
question - The example appears to be EngageBySailthru specific. Do we need a non-ST version if this is part of documenting how the SDK works? Or do we need this at all?

Reply:
This example is basically just a big code dump of how to call most of the available SDK functionality and receive callbacks, with the expectation that non-ST customers can just ignore the `EngageBySailthru` examples. I think this and our docs will be useful as a first reference, but we could always add an example app later if customers feel they're lacking context.